host/crypto/openssl/cert.c

```diff
@@ -774,6 +776,9 @@ oe_result_t oe_cert_get_rsa_public_key(
         EVP_PKEY_free(pkey);
     }
 
+    if (rsa)
```

Comment:
Is this even necessary? We don't actually use `rsa` anywhere and it looks like the variable can be eliminated in this function entirely.

Reply:
Good point, let me try just remove it.